kahuna/public/js/usage-rights/usage-rights-editor.js

```diff
@@ -88,6 +89,7 @@ usageRightsEditor.controller('UsageRightsEditorCtrl',
     }
 
     function save(data) {
+        ctrl.error = null;
```

Comment:
We receive errors from the server, which are rendered [here](https://github.com/guardian/grid/blob/master/kahuna/public/js/usage-rights/usage-rights-editor.html#L80).